network-configs/devnet-8/config.yaml

```diff
@@ -63,6 +63,8 @@ EJECTION_BALANCE: 31000000000
 MIN_PER_EPOCH_CHURN_LIMIT: 4
 # 2**16 (= 65,536)
 CHURN_LIMIT_QUOTIENT: 65536
+# [New in Deneb:EIP7514] 2**3 (= 8)
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
```

Comment:
can we change this on the fly?

Reply:
Yup, it'll be ignored by clients without it. Ideally we stop updating teku images, but sometimes we can't do that since we have to get the testnet to finalize. 

already made PRs and releases to ethereum-genesis-generator to include it for future testnets